scenes/menu/main_menu.gd

```diff
@@ -187,9 +201,11 @@ func _on_observe_game_started(data):
 
 	var player_deck_object = CardDefinitions.get_deck_from_str_id(player_deck_no_random)
 	var opponent_deck_object = CardDefinitions.get_deck_from_str_id(opponent_deck_no_random)
-	start_remote_game.emit(get_vs_info(player_name, player_deck_object, player_random_tag, opponent_name, opponent_deck_object, opponent_random_tag), start_data)
-
+	start_remote_game.emit(get_vs_info(player_name, player_deck_object, 
+		player_random_tag, opponent_name, opponent_deck_object, opponent_random_tag), start_data)
 
+# Accepts a signal from within the game scripts that a game has started
```

Comment:
This first sentence of the comment is very vague and not helpful. Of course the signal comes from scripts and of course the game_started function is when a game starts.
If you want to expand on what this signal is hooked up to, that's fine. Just don't add extra text that doesn't actually mean anything.